app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -50,28 +43,37 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
     $scope.page = newPage
     $scope.fetchProducts()
 
+  generateFilter = ->
+    filters = {}
+    for filter in productFilters
+      filters[filter] = $scope[filter] if $scope[filter]
+
+    filters
```

Comment:
I think you have forgotten to remove this method, it's now in the service :+1:

Reply:
Indeed, removed.